assig1/assignment-1/src/main/java/ro/utcn/spet/example/a1/controller/ConsoleController.java

```diff
@@ -0,0 +1,435 @@
+			return true;
+		}
+
+	}
+
+	private void handleStackUserList() {
+		stackUserManagementService.listStackUser().forEach(s -> print(s.toString()));
+	}
+
+	private void handleStackUserAdd() {
+		print("First name:");
+		String firstName = scanner.next().trim();
+		print("Last name:");
+		String lastName = scanner.next().trim();
+		StackUser stackUser = stackUserManagementService.addStackUser(firstName, lastName);
+		print("Created stackUser: " + stackUser + ".");
+	}
+
+	private void handleStackUserUpdateEmail() {
+		print("handleStackUserList ID:");
+		int id = scanner.nextInt();
+		print("Email address:");
+		String emailAddress = scanner.next().trim();
+		stackUserManagementService.updateEmailAddress(id, emailAddress);
+	}
+
+	private void handleStackUserRemove() {
+		print("handleStackUserList ID:");
+		int id = scanner.nextInt();
+		stackUserManagementService.removeStudent(id);
+	}
+
+
+	private void handleQuestionAdd(){
+		print("Title:");
+		String title = scanner.nextLine().trim();
+
+		print("Text:");
+		String text = scanner.nextLine().trim();
+		questionManagementService.addQuestion(stackUser.getId(), title, text);
+
+		print("Do you want to add a tag? [y]/[n]");
+		boolean role = false;
+		String option = scanner.nextLine().trim();
+		if(option.equals("y")){
+			role = true;
+		}
+		while(role){
+			print("Enter tag. Press [x] to exit." );
+			String tagName = scanner.nextLine().trim();
+			if(tagName.equals("x"))
+				role = false;
+			else{
+				tagManagementService.addTag(tagName);
+				tagQuestionManagementService.addTagQuestion(questionManagementService.filterTitle(title).get().getId(), tagManagementService.findTag(tagName).get().getId());
+			}
+		}
+
+
+		print("Created question.");
+
+	}
+
+	public void handleListAllTags(){
+		tagManagementService.findAll().forEach(s -> print(s.toString()));
+	}
+
+
+	public void handleQuestionRemove(){
+		print("Title:");
+		String title = scanner.nextLine().trim();
+		questionManagementService.remove(title);
+		print("Question removed");
+	}
+
+
+
+	private void handleFindAllQuestions(){
+		List<Question> reverseQuestion = questionManagementService.findAllQuestions();
+		Collections.reverse(reverseQuestion);
+		reverseQuestion.forEach(s -> print("VC: "+ calculateVoteCountQuestion(s) +" " + s.toString()));
+	}
+
+	private void handleFilterTitleQuestions(){
+		print("Title:");
+		int i =0;
+		String title = scanner.nextLine().trim();
+		Question question = questionManagementService.filterTitle(title).get();
+		print("VC: "+ calculateVoteCountQuestion(question) +" "+ questionManagementService.filterTitle(title).get().getText());
+		if(answerManagementService.findQuestionAllAnswers(question.getId()).size()  == 0){
+			print("For the moment there is no answer");
+		}
+		else {
+			List<Answer> filterTitle =  answerManagementService.findQuestionAllAnswers(question.getId());
+
+			//sort the answer list for a question with respect to VC
+			Collections.sort(filterTitle, new Comparator<Answer>() {
```

Comment:
You can simply replace that with `filterTitle.sort(Comparator.comparingInt(this::calculateVoteCountAnswer))`.